iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/BoundaryAdapter.java

```diff
@@ -47,16 +49,24 @@ public double getQ() {
         return getDelegate().getQ();
     }
 
-    @Override
+    /**
+     * @deprecated Not used anymore.
+     */
+    @Deprecated(since = "5.2.0")
     public Branch.Side getSide() {
-        return getDelegate().getSide() == null ? mergedSide : getDelegate().getSide();
+        throw new PowsyblException("Deprecated. Not used anymore");
     }
 
     @Override
     public Connectable getConnectable() {
         return getIndex().getConnectable(getDelegate().getConnectable());
     }
 
+    @Override
+    public DanglingLine getDanglingLine() {
+        return getIndex().getDanglingLine(getDelegate().getDanglingLine());
```

Comment:
Be careful if the dangling line is merged **in the merging view**. Put a different case for this.

Reply:
Done. Specific test has been added.